src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Button from './Button'
+
+describe('Button', () => {
+  
+  describe('common', () => {
+    const { container } = render(<Button>Найти</Button>)
+    const component = container.querySelector('.Button')
+    
+    it('renders correctly', () => {
+      expect(component).toHaveTextContent('Найти')
+    })
+
+    it('matches snapshot', () => {
+      expect(component).toMatchSnapshot()
+    })
+  })
+
+  describe('with extra className and disabled', () => {
+    const  { container } = render(<Button className={this.className} disabled={true}>Сохранить</Button>)
```

Comment:
Осталась проблема с классом. Тесты не работают.
```suggestion
    const  { container } = render(<Button className={this.className} disabled={true}>Сохранить</Button>)
```